tests/tests/test-block-gas.ts

```diff
@@ -32,3 +32,14 @@ describeDevMoonbeam("Block Gas - Limit", (context) => {
     );
   });
 });
+
+describeDevMoonbeam("Block Gas - Limit", (context) => {
+  // TODO: Joshy to fix block gas access in smart contract
+  it.skip("should be accessible within a contract", async function () {
```

Comment:
@JoshOrndorff , this is skipped for now. Unskip it once we have a working solution :)